src/components/ThreeViewer/Controls/DrawPVControl.jsx

```diff
@@ -75,3 +110,18 @@ const DrawPVControl = ({ middle, setPVPoints }) => {
 }
 
 export default DrawPVControl
+
+/**
+ * Compares two points, where one is an object and one is a list.
+ * The function allows a 1% deviation in each direction.
+ * @param {} p1 First Point as object with x,y,z attribute
+ * @param {} p2 Second point as list with three elements
+ * @returns
+ */
+function arePointsEqual(p1, p2) {
+  return (
+    Math.abs(p1.x - p2[0]) <= 0.01 * Math.abs(p1.x) &&
+    Math.abs(p1.y - p2[1]) <= 0.01 * Math.abs(p1.y) &&
+    Math.abs(p1.z - p2[2]) <= 0.01 * Math.abs(p1.z)
+  )
```

Comment:
Is relative deviation what we want here? As we center the building on x=0, y=0, this might lead to unexpected behaviour (though the functionality seems perfectly fine in testing).

Reply:
Should rather be euclidian distance smaller than some threshold I guess?

Reply:
The 3D coordinates should all be in meters, so that should be fine.